components/time/src/zone/mod.rs

```diff
@@ -245,8 +247,20 @@ impl TimeZoneInfo<models::Base> {
         TimeZone(tinystr::tinystr!(8, "utc")).with_offset(Some(UtcOffset::zero()))
     }
 
-    /// Sets a local time on this time zone.
-    pub const fn at_time(self, local_time: (Date<Iso>, Time)) -> TimeZoneInfo<models::AtTime> {
+    /// Sets the reference time for which time zone names should be resolved.
+    pub fn at_datetime(self, datetime: DateTime<Iso>) -> TimeZoneInfo<models::AtTime> {
+        self.at_inner(datetime.into())
+    }
+
+    /// Sets the reference time for which time zone names should be resolved.
+    pub fn at_minutes_since_local_epoch(
```

Comment:
the type is called `MinutesSinceEpoch`, so this should be `at_minutes_since_epoch`.